sematic/ui/src/pipelines/Artifacts.tsx

```diff
@@ -1,7 +1,7 @@
 import { Artifact } from "../Models";
 import Alert from "@mui/material/Alert";
 import AlertTitle from "@mui/material/AlertTitle";
-import { renderSummary } from "../types/Types";
+import { renderSummary } from "src/types/Types";
```

Comment:
Is this change intentional?

Reply:
We should favor root-relative imports over relative imports.